pipelines/creating-applications-with-cicd-pipelines.adoc

```diff
@@ -10,38 +10,34 @@ With {pipelines-title}, you can create a customized CI/CD solution to build, tes
 To create a full-fledged, self-serving CI/CD Pipeline for an application, you must perform the following tasks:
 
 * Create custom Tasks, or install existing reusable Tasks.
-* Create a Pipeline and PipelineResources to define the delivery Pipeline for your application.
+* Create and define the delivery Pipeline for your application.
+* Create a PersistentVolumeClaim attached to the Workspace to provide the volume or filesystem for pipeline execution.
 * Create a PipelineRun to instantiate and invoke the Pipeline.
 * Add Triggers to capture any events in the source repository.
 
 This section uses the `pipelines-tutorial` example to demonstrate the preceding tasks. The example uses a simple application which consists of:
 
-* A front-end interface `vote-ui`, with the source code in link:https://github.com/openshift-pipelines/vote-ui/tree/release-tech-preview-1[`ui-repo`] Git repository.
-* A back-end interface `vote-api`, with the source code in link:https://github.com/openshift-pipelines/vote-api/tree/release-tech-preview-1[`api-repo`]  Git repository.
-* `apply_manifest` and `update-deployment` Tasks in link:https://github.com/openshift/pipelines-tutorial/tree/release-tech-preview-1[`pipelines-tutorial`] Git repository.
+* A front-end interface `vote-ui`, with the source code in the link:https://github.com/openshift-pipelines/vote-ui/tree/release-tech-preview-2[`ui-repo`] Git repository.
+* A back-end interface `vote-api`, with the source code in the link:https://github.com/openshift-pipelines/vote-api/tree/release-tech-preview-2[`api-repo`]  Git repository.
```

Comment:
There should also be a comma after interface in these examples ("...front-end interface, `vote-ui`, with...").